package.json

```diff
@@ -33,7 +33,7 @@
     "test": "mocha"
   },
   "dependencies": {
-    "color-convert": "^0.5.3",
+    "color-convert": "^2.0.0",
```

Comment:
`color-convert` uses ES6 features that will break the core build, and it may also be not compatible with our current `chartjs-color` fork.

Reply:
Yeah. I discovered this as well and downgraded to 1.9.3 to fix that issue